src/Exception/MaintenanceModeActiveHttpException.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Exception;
+
+use Symfony\Component\HttpKernel\Exception\ServiceUnavailableHttpException;
+
+/**
+ * This exception is thrown when maintenance mode is active.
+ *
+ * @author Christian Schiffler <https://github.com/discordier>
+ */
+class MaintenanceModeActiveHttpException extends ServiceUnavailableHttpException
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(
+        // FIXME: should we initialize this to a sane default?
```

Comment:
Yeah, and that's why I quoted the RFC before which says every client that gets a 503 without a `Retry-After` header should treat this one just the same as a 500 :-)